From: Tony Jones <tonyj@suse.de>
Subject: Disable GSS options from config file
Upsteam: never

Disable GSS/Kerberos options from config file. They are disabled from configure
but need manual removal here.

---
 init.d/auditd.conf |    3 ---
 1 file changed, 3 deletions(-)

Index: audit-3.0.9/init.d/auditd.conf
===================================================================
--- audit-3.0.9.orig/init.d/auditd.conf
+++ audit-3.0.9/init.d/auditd.conf
@@ -30,8 +30,6 @@ tcp_max_per_addr = 1
 ##tcp_client_ports = 1024-65535
 tcp_client_max_idle = 0
 transport = TCP
-krb5_principal = auditd
-##krb5_key_file = /etc/audit/audit.key
 distribute_network = no
 q_depth = 2000
 overflow_action = SYSLOG
